cmake/sphinx.cmake

```diff
@@ -151,7 +151,7 @@ macro(ADD_SPHINX_DOCUMENTATION)
   # All parameters
 
   # Get the resource folder
-  get_resource_folder(mpi_cmake_modules MPI_CMAKE_MODULES_RESOURCES_DIR)
+  # get_resource_folder(mpi_cmake_modules MPI_CMAKE_MODULES_RESOURCES_DIR)
```

Comment:
Will this be needed again in the future?  Otherwise I would prefer to remove it completely.

Reply:
Oo I need to check if the documentation is compiled properly...

Reply:
You do not need it because of this: https://github.com/machines-in-motion/mpi_cmake_modules/blob/mnaveau/remove_ament/Config.cmake.in#L18
You directly export the path to the resources in the `${PROJECT_NAME}Config.cmake`
So whenever someone does a `find_package(mpi_cmake_module)`, cmake will set the `MPI_CMAKE_MODULES_RESOURCES_DIR`.

Reply:
BTW I will create the additional doc we need.